lib/delayed_job_worker_pool/registry.rb

```diff
@@ -0,0 +1,51 @@
+module DelayedJobWorkerPool
+  # Keeps track of worker groups and their workers.
+  class Registry
+    class GroupAlreadyExists < StandardError; end
+    class GroupNotFound < StandardError; end
+
+    def initialize
+      @groups = {}
+    end
+
+    def include?(pid)
+      worker_pids.include?(pid)
+    end
+
+    def empty?
```

Comment:
Similar comment as `include?` i.e. it wasn't clear to me if this was checking groups or workers. Should we call this `has_workers?` to make the intent more obvious?